.github/workflows/docker-build-auth.yml

```diff
@@ -0,0 +1,78 @@
+name: Publish experimental auth Docker image
```

Comment:
couldn't you just make this generic? like `experimental` with a commit hash?

Reply:
Not sure what you have in mind. Generic as in not auth-related?